src/pages/resultsView/ResultsViewPageStore.ts

```diff
@@ -624,12 +624,30 @@ export class ResultsViewPageStore {
 
     @observable public urlValidationError: string | null = null;
 
-    @computed get profileFilter() {
+    @computed get profileFilterSet() {
         if (this.urlWrapper.query.profileFilter) {
-            return parseInt(this.urlWrapper.query.profileFilter, 10);
-        } else {
-            return 0;
+            if (isNaN(parseInt(this.urlWrapper.query.profileFilter, 10))) {
+                return stringListToSet(
+                    this.urlWrapper.query.profileFilter.split(',')
+                );
+            }
+        }
+        return undefined;
+    }
+
+    @computed get dataPriority() {
```

Comment:
i thought we were going to get rid of dataPriority and treat it as an alias to provide backward compatibility?

Reply:
yes, this is for backward compatibility